### PR TITLE
Add testing to TravisCI

### PR DESCRIPTION
**Pull request by MichaelWasher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,41 @@ go:
...
+
+    - name: "StaticTCPDump Test"
+      script:
+        - make static-tcpdump
+    
 before_deploy:
   - make package
-
+  
 deploy:
```
Comment (MichaelWasher) [inline .travis.yml:42]: deploy needs to be placed into a stage otherwise this is being run at the end of every build.

Comment (MichaelWasher) [inline .travis.yml:42]: This need to be placed into a stage otherwise this is being run on every build in the matrix.

**Comment by bostrt** [comment]
I see the following in travis logs. Would you mind trying to address that? 

https://travis-ci.org/github/eldadru/ksniff/jobs/765901455#L299-L301
```
$ make test
Error: unknown flag: --short
Run 'kubectl help' for usage.
```

**Comment by MichaelWasher** [comment]
It worked out in KinD when I tested it locally. I've also noted that when I ran `make test` in TravisCI without the kubectl / KinD additions it failed in TestComplete_PodNameSpecified due to the missing kube-context.
```
?   	ksniff/kube	[no test files]
Usage:
--- FAIL: TestComplete_PodNameSpecified (0.00s)
    sniff_test.go:55: 
        	Error Trace:	sniff_test.go:55
        	Error:      	Expected nil, but got: context doesn't exist
        	Test:       	TestComplete_PodNameSpecified
FAIL
```
https://www.travis-ci.com/github/MichaelWasher/ksniff/jobs/495064017

I didn't look too deeply, but it might not require access to the cluster but it does look like it's dependant on a kube-context being present somewhere like `~/.kube/config`. I guess we could emulate that if it's not there, but I figured this would be easier and it doesn't add too much time on testing.

I'll see if I can write a short integration test(s) as well. Do you want it written in Go, or something like Python/Bash to keep it separate from the unit tests and perform it on the compiled binary?

**Comment by bostrt** [comment]
Closing this since we're testing in GitHub Actions now.

**Pull request closed without merging**